src/navigate-pop/switching-apps.md

```diff
@@ -0,0 +1,50 @@
+# Navigate Between Windows and Applications
+
+Pop!\_OS uses the GNOME desktop environment, which includes keyboard shortcuts for quickly switching between system and application windows.
+
+---
+
+## Navigate Between Windows
+
+View all windows in the current workspace by pressing `SUPER` + `D`. For more information about Workspaces, see [Using Workspaces](using-workspaces.md)
+
+![Display All Windows](/images/locate-launch-applications/display-all-windows.png)
+
+Use the the `🠐` `🠑` `🠒` `🠓` keys or mouse to move between windows. Select a window with left-click or by pressing `Enter`.
+
+![Select Displayed Window](/images/locate-launch-applications/select-displayed-window.png)
+
+Switch back to the previously selected window by pressing `Alt` + `Esc`.
+
+### Navigate Windows with Keyboard Shortcuts
+
+| Shortcut             | Action |
+|----------------------|--------|
+| <ul><li>SUPER + 🠓</li><li>SUPER + Keypad down</li><li>SUPER + J | Switch focus to the below window. |
+| <ul><li>SUPER + 🠐</li><li>SUPER + Keypad left</li><li>SUPER + H | Switch focus to the left window. |
+| <ul><li>SUPER + 🠒</li><li>SUPER + Keypad right</li><li>SUPER + L | Switch focus to the right window. |
+| <ul><li>SUPER + 🠑</li><li>SUPER + Keypad up</li><li>SUPER + K | Switch focus to the above window. |
+
+
+## Navigate Between Running Applications
+
+Pressing `SUPER` + `Tab` and `Alt` + `Tab` will display a list of running application icons across the screen. Continue holding `SUPER` or `Alt` and pressing `Tab` to toggle between applications. You can also navigate between applications with the `🠐` `🠒`  keys.
+
+![Toggle Apps](/images/locate-launch-applications/toggle-apps.png)
+
+Mouse over or release `tab` on highlighted application icons to display multiple open windows for that application. Continue holding `SUPER` or `Alt` and press ` to toggle between the multiple windows.
```

Comment:
Capitalize `Tab` to make it consistent with everywhere else. (Actually, there's a `Tab` above and a `TAB` below, so make them all consistent.)